audiblelight/config.py

```diff
@@ -13,6 +13,7 @@
 # Scene
 SCENE_DURATION = 60
 REF_DB = -65
```

Comment:
should this line be removed since the min and max variables below indicate that this will be drawn from a distribution?